sqladmin/models.py

```diff
@@ -954,6 +943,20 @@ def get_export_columns(self) -> List[str]:
             defaults=self._list_prop_names,
         )
 
+    def _get_sort_fields(self, request: Request) -> List[Tuple[str, bool]]:
```

Comment:
I think this method can be inlined in the `sort_query`, or at least be public to be used so overriding the sort_query is made easy. But I think it's small enough to fit into the public `sort_query`.

Reply:
Do you mean something like that? Only one method.
```py
    def sort_query(self, stmt: Select, request: Request) -> Select:
        """
        A method that is called every time the fields are sorted
        and that can be customized.
        By default, sorting takes place by default fields.
        """
        sort_by = request.query_params.get("sortBy", None)
        sort = request.query_params.get("sort", "asc")

        if sort_by:
            sort_fields = [(sort_by, sort == "desc")]
        else:
            sort_fields = self._get_default_sort()

        for sort_field, is_desc in sort_fields:
            if is_desc:
                stmt = stmt.order_by(desc(sort_field))
            else:
                stmt = stmt.order_by(asc(sort_field))

        return stmt
```